pkg/generate/app/templatelookup.go

```diff
@@ -92,7 +100,8 @@ func (r *TemplateFileSearcher) Search(terms ...string) (ComponentMatches, error)
 			Object()
 
 		if err != nil {
-			return nil, err
+			glog.V(5).Infof("Error from template search: %v", err)
+			continue
```

Comment:
not aborting on an error seems like a mistake. or at least printing it as a warning.  Doesn't this mean if the template file the user intended to use has a typo, you're going to silently skip over that file?